src/go/configgenerator/listener_generator_test.go

```diff
@@ -605,32 +615,68 @@ func TestBackendAuthFilter(t *testing.T) {
 }
 `,
 		},
+		{
+			desc:             "Fail when invalid dependency error behavior is provided",
+			depErrorBehavior: "UNKNOWN_ERROR_BEHAVIOR",
+			fakeServiceConfig: &confpb.Service{
+				Name: testProjectName,
+				Apis: []*apipb.Api{
+					{
+						Name: "testapi",
+					},
+				},
+				Backend: &confpb.Backend{
+					Rules: []*confpb.BackendRule{
+						{
+							Selector:        "testapipb.bar",
+							Address:         "https://testapipb.com/foo",
+							PathTranslation: confpb.BackendRule_CONSTANT_ADDRESS,
+							Authentication: &confpb.BackendRule_JwtAudience{
+								JwtAudience: "bar.com",
+							},
+						},
+					},
+				},
+			},
+			wantError: "unknown value for DependencyErrorBehavior (UNKNOWN_ERROR_BEHAVIOR), accepted values are: map[0:UNSPECIFIED 1:BLOCK_INIT_ON_ANY_ERROR 2:ALWAYS_INIT]",
```

Comment:
It is not clear to:me:   should it "2:ALWAYS_INIT"  or just "ALWAYS_INIT"?

Reply:
Should be ALWAYS_INIT, fixed

Reply:
Done